### PR TITLE
Fix a flaky test

### PR DESCRIPTION
**Pull request by mikhailOK**

### CODE REVIEW

**Comment by nearmax** [comment]
Is the integration test fix coming in a separate PR?

**Comment by mikhailOK** [comment]
integration tests pass on my machine, haven't tested on CI

**Comment by nearmax** [comment]
CI machines are weaker than our laptops. You can rerun integration tests on CI here: https://gitlab.com/nearprotocol/nearcore/pipeline_schedules

File: node/alphanet/src/lib.rs [hunk 1]
```diff
@@ -195,11 +195,14 @@ mod tests {
...
         bob.start();
 
-        // Wait until alice and bob produce at least one block.
+        // Wait until alice and bob produce at least 4 blocks.
+        // This condition is a hack: they produce 0-2 empty blocks
+        // before bob gets the tx gossip from alice,
+        // then one with transaction and one with receipt
         wait(
             || {
-                alice.as_thread_ref().client.shard_client.chain.best_index() >= 3
```
Comment (nearmax) [inline node/alphanet/src/lib.rs]: This should wait by polling final transaction result, instead.

**Pull request merged**